### ISSUE TITLE
DataTable : Edit, save and cancel button do not have accessible names 

### ISSUE THREAD

**Issue by ankitprahladsoni** [primefaces/primereact#2128]
**I'm submitting a ...**  (check one with "x")
```
[x] feature request
```
I was trying to write some test cases for the table editing using `@testing-library/react`, but I could not locate the edit, save or cancel button on the row because they don't have accessible names that are needed for testing library. Such as `screen.getByRole("button",{name:"edit"})`

**Current behavior**
Edit, save and cancel button during row editing on Datatable do not have accessible names

**Expected behavior**
Edit, save and cancel button during row editing on Datatable should have accessible names

**Minimal reproduction of the problem with instructions**
RowEdit example on  https://www.primefaces.org/primereact/showcase/#/datatable/edit
